reactor/presence.sls

```diff
@@ -1,6 +1,8 @@
+# actions to run when a minion disappears or appears in the master
+
 # remove unused keys for minions that are not present anymore
-{% for minion_id in data['lost'] %}
-remove_unused_keys_for_{{ minion_id }}:
-   wheel.key.delete:
-     - match: {{ minion_id }}
-{% endfor %}
+# {% for minion_id in data['lost'] %}
```

Comment:
Long Term: We should probably have a UI to remove old node.
Short Term: Auto remove is probably OK - leaving "dead nodes" around will probably cause us issues when doing things like updates